### DISCUSSION TITLE
How do you maintain discussion section in your repo

### DISCUSSION THREAD

**Discussion by Ananyapk25** [community/community#63171] [Discussions]
### Select Topic Area

Question

### Body

Discussion repo usage

**Comment by siliconvolley** [comment]
is meant for discussing any new features or issues in the projects

**Comment by shettyvarshaa** [comment]
yeah its for discussing anything about your code

**Comment by queenofcorgis** [comment]
GitHub's [Acceptable Use Policies](https://docs.github.com/en/site-policy/acceptable-use-policies/github-acceptable-use-policies) prohibits coordinated or inauthentic activity like rapid questions and answers. As a result, we'll be unmarking the answer and locking this post. Please note any future violations may result in a temporary or indefinite block from the Community. Thanks for understanding.
